selendroid-standalone/src/main/java/io/selendroid/standalone/android/impl/AbstractDevice.java

```diff
@@ -596,4 +597,8 @@ public boolean equals(Object o) {
   public int hashCode() {
     return device.hashCode();
   }
+
+  public String getDeviceType() {
+    return deviceType;
```

Comment:
`device` for selendroid is either an emulator or real device.

based on 2. I'd prefer to use model for emulators as well and keep the terminology clear.
Implementation part is going to be reduced this way as well.